tvb/simulator/models/wong_wang_exc_io_inh_i.py

```diff
@@ -0,0 +1,309 @@
+        doc="""[ms]. Inhibitory population NMDA decay time constant.""",
+        order=13)
+
+    J_i = arrays.FloatArray(
+        label=r":math:`J_{i}`",
+        default=numpy.array([1.0, ]),
+        range=basic.Range(lo=0.001, hi=2.0, step=0.001),
+        doc="""[nA] Local inhibitory current""",
+        order=14)
+
+    W_i = arrays.FloatArray(
+        label=r":math:`W_i`",
+        default=numpy.array([0.7, ]),
+        range=basic.Range(lo=0.0, hi=1.0, step=0.01),
+        doc="""Inhibitory population external input scaling weight""",
+        order=15)
+
+    I_o = arrays.FloatArray(
+        label=":math:`I_{o}`",
+        default=numpy.array([0.382, ]),
+        range=basic.Range(lo=0.0, hi=1.0, step=0.001),
+        doc="""[nA]. Effective external input""",
+        order=16)
+
+    G = arrays.FloatArray(
+        label=":math:`G`",
+        default=numpy.array([2.0, ]),
+        range=basic.Range(lo=0.0, hi=10.0, step=0.01),
+        doc="""Global coupling scaling""",
+        order=17)
+
+    lamda = arrays.FloatArray(
+        label=":math:`\lambda`",
+        default=numpy.array([0.0, ]),
+        range=basic.Range(lo=0.0, hi=1.0, step=0.01),
+        doc="""Inhibitory global coupling scaling""",
+        order=18)
+
+    state_variable_range = basic.Dict(
+        label="State variable ranges [lo, hi]",
+        default={"S_e": numpy.array([0.0, 1.0]), "S_i": numpy.array([0.0, 1.0])},
+        doc="Population firing rate",
+        order=22
+    )
+
+    variables_of_interest = basic.Enumerate(
+        label="Variables watched by Monitors",
+        options=['S_e', 'S_i'],
+        default=['S_e', 'S_i'],
+        select_multiple=True,
+        doc="""default state variables to be monitored""",
+        order=23)
+
+    state_variables = ['S_e', 'S_i']
+    _nvar = 2
+    cvar = numpy.array([0], dtype=numpy.int32)
+
+    def configure(self):
+        """  """
+        super(ReducedWongWangExcIOInhI, self).configure()
+        self.update_derived_parameters()
+
+    def _numpy_dfun(self, state_variables, coupling, local_coupling=0.0):
+        r"""
+        Equations taken from [DPA_2013]_ , page 11242
+
+        .. math::
+                 x_{ek}       &=   w_p\,J_N \, S_{ek} - J_iS_{ik} + W_eI_o + GJ_N \mathbf\Gamma(S_{ek}, S_{ej}, u_{kj}),\\
+                 H(x_{ek})    &=  \dfrac{a_ex_{ek}- b_e}{1 - \exp(-d_e(a_ex_{ek} -b_e))},\\
+                 \dot{S}_{ek} &= -\dfrac{S_{ek}}{\tau_e} + (1 - S_{ek}) \, \gammaH(x_{ek}) \,
+
+                 x_{ik}       &=   J_N \, S_{ek} - S_{ik} + W_iI_o + \lambdaGJ_N \mathbf\Gamma(S_{ik}, S_{ej}, u_{kj}),\\
+                 H(x_{ik})    &=  \dfrac{a_ix_{ik} - b_i}{1 - \exp(-d_i(a_ix_{ik} -b_i))},\\
+                 \dot{S}_{ik} &= -\dfrac{S_{ik}}{\tau_i} + \gamma_iH(x_{ik}) \,
+
+        """
+        S = state_variables[:, :]
+        S[S < 0] = 0.
+        S[S > 1] = 1.
+
+        c_0 = coupling[0, :]
+
+        # if applicable
+        lc_0 = local_coupling * S[0]
```

Comment:
@dionperd this local coupling is between excitatory populations, which may be not be correct. Please see my comment [here](lateral connectivity is frequently regarded as between inhibitory interneurons, but one could want to have any mix of the two. Please look at how it is done in the Wilson Cowan (I don't remember right now) and do similarly here.) on Julie's PR.